src/luarocks/core/cfg.lua

```diff
@@ -492,8 +494,17 @@ local function make_defaults(lua_version, target_cpu, platforms, home)
       defaults.variables.LIBFLAG = (os.getenv("LDFLAGS") or "").." -shared"
    end
 
+   if platforms.dragonfly then
+      defaults.arch = "dragonfly-"..target_cpu
+      defaults.variables.CC = os.getenv("CC") or "cc"
+      defaults.variables.CFLAGS = os.getenv("CFLAGS") or defaults.variables.CFLAGS
+      defaults.variables.LD = defaults.variables.CC
+      defaults.variables.LIBFLAG = (os.getenv("LDFLAGS") or "").." -shared"
+   end
+
    if platforms.openbsd then
       defaults.arch = "openbsd-"..target_cpu
+      defaults.gcc_rpath = false
```

Comment:
why turn this off?

Reply:
probably because OpenBSD has switched to clang instead of gcc? Back in 2014 I made a similar change for FreeBSD because of this, at least: b66bc076af602d5afe1eb6f2bf0275d3e9bc2422